adb_graphics/figure_builders.py

```diff
@@ -0,0 +1,280 @@
+# pylint: disable=invalid-name
```

Comment:
This new file is a collection of old functions that build the skewt and map plots, plus new function that helps set the figure parameters and add the observation panel for composite reflectivity.